framework/packages/abstract-app/Cargo.toml

```diff
@@ -39,6 +39,5 @@ workspace-hack = { version = "0.1", path = "../../workspace-hack" }
 
 [dev-dependencies]
 cosmwasm-schema = { workspace = true }
-speculoos = { workspace = true }
```

Comment:
Also did small sidequest, hehe

Reply:
Don't we like speculoos ? @CyberHoward

Reply:
We discussed it already when you were gone, but there is no benefit for using it. It's also not maintained and no one uses it.